stories/translator/doc-elements.tsx

```diff
@@ -80,6 +80,9 @@ export const DocElement = () => {
     // EVENT HANDLERS
     // =========================================================================
     const handleFormBuilderButton = () => {
+        if (formBuilderRef.current && schemaProvided) {
```

Comment:
will a truthy check suffice? what's the use of tracking `schemaProvided` in a state btw?

```suggestion
        if (formBuilderRef.current && schema) {
```